src/github_sdk.py

```diff
@@ -71,19 +71,29 @@ def send_envelope(trace):
 def get_extra_metadata(job):
```

Comment:
All of the changes in this file can more easily be view if you look at the `trace.json` changes.

The changes in here are to make the function `_generate_trace` a lot simpler. For instance, now, I can just do this:

```python
    transaction["tags"] = meta["tags"]
    transaction["contexts"]["trace"]["data"] = meta["data"]
```